src/usr/local/www/widgets/widgets/system_information.widget.php

```diff
@@ -504,7 +531,7 @@ function systemStatusGetUpdateStatus() {
 		},
 		dataType: 'html',
 		success: function(data){
-			$('[id^=widget-system_information] #updatestatus').html(data);
+			$('#system-information-widget-updatestatus').html(data);
```

Comment:
There is now the possibility of having multiple copies of the System Information Widget on the dashboard. But actually the code that updates the values is organized to only be running once in the background (look for uses of $widget_first_instance). So the expression here needs to update all instances of the relevant widget(s). That is what the id^= stuff was doing. I traveling and not able to test right now. But it needs to be tried with multiple DSys Info widgets open with some items duplicated in each.